README.md

```diff
@@ -30,6 +30,20 @@ We could use some help with:
 
 -   (install pip)
 -   **pip install sphinx**
+- pip install sphinx_rtd_theme
+- then in source/conf.py
```

Comment:
We should decide: is the documentation by default using the rtd theme?
If yes then we can change the `conf.py` accordingly and not asking the user to make this change.
If no, we can add it as an optional step if the user wants to build it with the rtd theme.

in any case this part should be inside a code tag 

```bash
pip install sphinx sphinx_rtd_theme
```
etc.

Note that for building the RTD online it is not necessary to specify the rtd theme, it can be as we have it today `default`, it will automatically override to sphinx_rtd_theme

Reply:
IMO, it would be better to enable RTD theme by default.

Reply:
so let's modify the conf.py